docs/develop/howtos/restrict_access.md

```diff
@@ -0,0 +1,52 @@
+# How to restrict access to pages
+
+Sometimes it can be desirable to live by the motto "better safe than sorry", especially regarding potentially sensitive features like the administration panel (enabled in InvenioRDM v12).
+
+This guide briefly describes how to narrow down access to subsets of the system.
+
+
+## Restricting access for IP ranges via `nginx`
+
+While most features in InvenioRDM are guarded by configurable permission policies, this isn't necessarily always the case.
+For these exceptions, as well as extra precautions generally, it can be beneficial to restrict access on an `nginx` level.
+
+!!! info "Current exceptions"
+    At the time of writing, one of these exceptions is the administration panel which has a hard-coded check for the `administration-access` action.
+
+An access restriction based on the client's IP address can be put into place via the `nginx` configuration, e.g. by adding nested `location` directives in the existing configuration:
+
+```nginx
+location / {
+    uwsgi_pass ui_server;
+    include uwsgi_params;
+    # ... your configuration for the UI paths ...
+
+    # restrict access to the administration panel UI to TU Wien network
+    location /administration/ {
+      # action directives like `uwsgi_pass` aren't inherited like other configs
+      uwsgi_pass ui_server;
+
+      # allow TUW networks (according to the RIPE database)
```

Comment:
Is this the correct URL to check?
```suggestion
      # allow TUW networks (according to the RIPE database)
      # see: https://apps.db.ripe.net/db-web-ui/query
```

Reply:
yes, that's the base; the full URL would be https://apps.db.ripe.net/db-web-ui/query?bflag=false&dflag=false&inverse=origin&rflag=true&searchtext=AS679&source=RIPE
but this was sent to me by our network guys, and i have no clue why we're listed as AS679

i'm also not entirely sure if this link is really interesting for others, maybe we could even delete the remark in brackets?